### ISSUE TITLE
string mutation causes UnicodeDecodeError

### ISSUE THREAD

**Issue by golankai** [boxed/mutmut#253]
I am working on a project that includes text normalization and using mutmut for mutation testing.
mutmut is mutating line 35 to be `normalized = 'XXXX'` instead of `normalized = ''`
![image](https://user-images.githubusercontent.com/72255040/181268587-447a1723-ea12-41bf-8442-50d2cb0ba7f9.png)

I then get the following error:
![image](https://user-images.githubusercontent.com/72255040/181268922-625c0cdf-8f80-4265-aa2a-c33ff4da283b.png)

when I run my tests regularly they work, and when I apply this change manually and run my tests, they work too.

Have any Idea what can happen that causes this failure?

**Comment by boxed** [comment]
Are you sure this is the mutation that fails? Is that what you get with `mutmut show 0`?

"position 35" in the error would be the column 35, not the line 35 of the file.

**Comment by golankai** [comment]
the position is indeed 35, but coincidently also the row I referred to :) when reporting the error, the position changes, as it is using a random string from Hypothesis testing.
I am sure.

1. Running the tests themselves produces no error.
2. Applying the mutation manually and running the tests produces an assertion error (this is good) but not a fatal Error of 'utf-8'.
3. running `mutmut run` crashes as I showed above, and then also by running `mutmut run 4` (4 is the id of this mutant).

This is quite weird. I have seen before in our project those errors that comes from normalization, but I don't get how and why this mutant causes them, only when running through mutmut.
